<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TeaBank</class>
 <widget class="QWidget" name="TeaBank">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="mainLabel">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>10</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;缴费系统&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exitButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>260</y>
     <width>80</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>31</x>
     <y>61</y>
     <width>321</width>
     <height>181</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="learnFeeLabel">
        <property name="text">
         <string>本学年学费：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="learnFeeLineEdit"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="residenceLabel">
        <property name="text">
         <string>本学年住宿费：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="residenceLineEdit"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QCheckBox" name="lifeFeeCheckBox">
        <property name="text">
         <string>水电网费缴纳</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="learn_residenceCheckBox">
        <property name="text">
         <string>学宿费缴纳</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="commitButton">
        <property name="text">
         <string>发布</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="reButton">
        <property name="text">
         <string>重置水电缴费</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
